components/scream/src/physics/share/physics_functions.hpp

```diff
@@ -98,6 +98,19 @@ struct Functions
   //checks temperature for negatives and NaNs
   KOKKOS_FUNCTION
   static void check_temperature(const Spack& t_atm, const char* func_name, const Smask& range_mask);
+
+  // Computes exner function and potential temperature given the mid-level pressure
```

Comment:
I think it would be nice to have a description of each of these "universal" functions with a bit more detail, including any assumptions about units, and the role of `range_mask`, since they could be used in various contexts.

Reply:
Good point, Peter C suggested I break off the physics functions part into its own PR (see #805).  I'll make these changes there.

Reply:
handles in #805